src/libraries/System.Net.HttpListener/tests/TrimmingTests/CookieToServerStringTest.cs

```diff
@@ -0,0 +1,308 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CookieToServerStringTest
+{
+    /// <summary>
+    /// Tests that the System.Net.CookieExtensions.ToServerString()
+    /// method works as expected when used in a trimmed application.
+    /// </summary>
+    internal class Program
```

Comment:
Is this a minimal program to test this scenario? It seems like a lot of code, can any of it be removed?

Reply:
Yup, I adapted much of this from existing tests. I've trimmed out unneeded parts.